wfl/App_Start/WebApiConfig.cs

```diff
@@ -1,17 +1,21 @@
 ﻿using System.Web.Http;
+using Drum;
+using Hypermedia;
+using Hypermedia.Transforms;
+using Hypermedia.Siren;
+using Symphono.Wfl.Models;
 
 namespace Symphono.Wfl
 {
     public static class WebApiConfig
     {
         public static void Register(HttpConfiguration config)
         {
-            config.MapHttpAttributeRoutes();
+            config.EnableHypermedia()
+                .RegisterDefaultTransform(new ReflectiveTransform());
```

Comment:
If there is no entity specific transform selected for an entity, it'll use the reflective transform.  The reflective transform simply looks over all public properties of an object and returns them as properties.  It then also adds a class based on the class name I believe.  I'm guessing this was added mostly as a "just in case" we don't have a profile registered?